components/voie-information.js

```diff
@@ -0,0 +1,114 @@
+import {useState} from 'react'
+import PropTypes from 'prop-types'
+import {ChevronUp, ChevronDown} from 'react-feather'
+import theme from '@/styles/theme'
```

Comment:
Il manque un saut de ligne.

```suggestion
import {ChevronUp, ChevronDown} from 'react-feather'

import theme from '@/styles/theme'
```